chart/templates/configmap.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: job-service-config
+  namespace: {{ .Release.Namespace }}
```

Comment:
Is this required? AFAIK you don't need to specify the namespace here, it's implied by `helm install -n <NAMESPACE>`.
@pchila please correct me if I'm wrong :D

Reply:
No it's not required, helm will generate the appropriate metadata for the target namespace. Not sure why I put it there ...